templates/controllers/grid/users/chapter/form/chapterForm.tpl

```diff
@@ -37,6 +37,16 @@
 	{fbvElement type="textarea" name="abstract" id="abstract" value=$abstract  rich="extended" multilingual=true}
 	{/fbvFormSection}
 
+	{fbvFormSection title="metadata.property.displayName.pages" for="customExtras"}
+	{fbvElement type="text" id="pages" value=$pages inline=true size=$fbvStyles.size.LARGE}
+	{/fbvFormSection}
+
+	{if $IsOnlineFirstSubmission}
```

Comment:
We use `lowerCamelCase` conventions for Smarty/PHP variables -- the first `I` should be lower case.